src/jdk.attach/windows/classes/sun/tools/attach/AttachProviderImpl.java

```diff
@@ -34,19 +34,11 @@
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
+import jdk.internal.util.Architecture;
+
```

Comment:
Not removed in [commit `ab084c2` or `52ca4a7`](https://github.com/openjdk/jdk/pull/13357/files/105ce60528f7ef84e31ac3db64e6eb63860b4a70..52ca4a70fc3de9e285964f9545ea8cd54e2d9924):
```suggestion
```